### PR TITLE
Form to claim paper ownership with a password

### PR DESCRIPTION
**Pull request by bdc34**
Implements form to allow a user to claim ownership of a paper with a password.

On success this will write to the arXiv_paper_owners table.

https://arxiv-org.atlassian.net/browse/ARXIVD-51

### Empty form
Selection_071
### Bad password
Selection_072
### Already ownes
Selection_073
### Success
Selection_076
### Invalid form
Selection_075

### CODE REVIEW

**Comment by jimentwood** [comment]
I am confused by the user name. It appears to be displaying first_name last_name in the screenshots. And also includes unicode. Is that intentional?

**Comment by jimentwood** [comment]
Ideally this page would have a clear link back to the user dashboard
https://arxiv.org/user

That gives the user the link to verify and act on the paper.

...I proposed a change to add that link

File: admin_webapp/templates/ownership/need_paper_password.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+{% endif %}
+
+
+
+<div>
+  By entering the paper ID and <i>paper password</i> associated with a
+  paper, you can become registered as the <i>owner</i> of a paper.
+  Being the owner of a paper confers a number of benefits: for
+  example, you can make changes to the paper, cross-list the paper and
+  add a journal reference.
```
Comment (jimentwood) [inline admin_webapp/templates/ownership/need_paper_password.html:44]:
```suggestion
  add a journal reference. <br> Your <a href="/user">user dashboard</a> is where you can view and update papers you own. 
```

**Comment by jimentwood** [comment]
How about a sentence at the beginning.

This form allows you to link

**Comment by bdc34** [comment]
> I am confused by the user name. It appears to be displaying first_name last_name in the screenshots. And also includes unicode. Is that intentional?

It is displaying first_name last_name and the unicode is displayed incorrectly. In arxiv my user account has the name "Brian Caruso-Wardęga" which includes a Polish name an arXiv user. I added that to know where unicode names work and don't work.

Do you want something other than first_name last_name?

File: admin_webapp/templates/ownership/need_paper_password.html (hunk 1 above)
Comment (bdc34) [reply admin_webapp/templates/ownership/need_paper_password.html:44]: Link added.

**Comment by bdc34** [comment]
> How about a sentence at the beginning.
> 
> This form allows you to link

I've added a sentence like that. Feel free to propose one as a commit or here in a comment and I'll add your text.

**Comment by jimentwood** [comment]
For the name/username part. I would like us to not use username and just the full name and email.
(this fits into a broader strategy of depreciating use of usernames which overtime may not represent a person's identity... and won't be that useful if we go with ORCID and possibly other authentication systems)

Suggest this section
```
I certify that my name is <b>{{request.auth.user.display_name}}</b>, my username
    is <b>{{request.auth.user.name.forename}} {{request.auth.user.name.surname}}</b>
```

change to

```
I certify that my name is <b>{{request.auth.user.name.forename}} {{request.auth.user.name.surname}}</b>
```

**Comment by bdc34** [comment]
Here is the form with Jim's suggestions:

Selection_077

**Review by jimentwood — APPROVED** [review]
looks good

**Pull request merged**